src/test/logic/token/utils/tokenHelpers.test.ts

```diff
@@ -0,0 +1,193 @@
+import logo from 'src/assets/icons/icon_etherTokens.svg'
+import { makeToken } from 'src/logic/tokens/store/model/token'
+import {
+  ETH_ADDRESS,
+  getERC20DecimalsAndSymbol,
+  getEthAsToken, isERC721Contract,
+  isTokenTransfer,
+} from 'src/logic/tokens/utils/tokenHelpers'
+import { getMockedTxServiceModel } from 'src/test/utils/safeHelper'
+
+
+describe('getEthAsToken', () => {
+  it('It should return the balance as eth token if given eth balance returns', () => {
+    // given
+    const balance = '1000'
+    const expectedResult = makeToken({
+      address: ETH_ADDRESS,
+      name: 'Ether',
+      symbol: 'ETH',
+      decimals: 18,
+      logoUri: logo,
+      balance,
+    })
+
+    // when
+    const result = getEthAsToken(balance)
+
+    // then
+    expect(result).toStrictEqual(expectedResult)
+  })
+})
+
+describe('isTokenTransfer', () => {
+  const safeAddress = '0xdfA693da0D16F5E7E78FdCBeDe8FC6eBEa44f1Cf'
+  it('It should return true if the transaction has no value but has transfer data',  () => {
+    // given
+    const transaction = getMockedTxServiceModel({ to: safeAddress, value: '0', data: '0xa9059cbb' })
+    const expectedResult = true
+    // when
+    const result = isTokenTransfer(transaction)
+
+    // then
+    expect(result).toEqual(expectedResult)
+  })
+  it('It should return false if the transaction has no value but has data different from transfer',  () => {
```

Comment:
`It should return false if the transaction has no value but and no "transfer" function signature encoded in data`